src/isolate_create_params.rs

```diff
@@ -187,6 +191,10 @@ pub(crate) mod raw {
     pub only_terminate_in_safe_scope: bool,
     pub embedder_wrapper_type_index: int,
     pub embedder_wrapper_object_index: int,
+    pub cpp_heap_params: SharedPtr<CppHeapCreateParams>,
+    // This is an std::vector<std::string>. It's usually no bigger
+    // than three or four words but let's take a generous upper bound.
```

Comment:
Just wanted to point out that this is only safe because the CreateParams struct is always "stored" in rust and "loaded" in C++. If it were the other way around you'd want to make it a "generous lower bound".